### PR TITLE
Update README-building-wheels.md

### PR DESCRIPTION
**Pull request by glennmoy**
Updates `python/README-building-wheels.md` with further instructions for beacon developers.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205510215892647
  - https://app.asana.com/0/0/1205549170998772

### CODE REVIEW

File: python/README-building-wheels.md [hunk 1]
```diff
@@ -6,16 +6,37 @@ repository, including both changes to tracked files, and ANY untracked files.
...
 Inside the root directory (i.e., one level above this python directory), run
 
 ```
-docker run -e TRAVIS_COMMIT=<commit_number_to_use> --rm -w /ray -v `pwd`:/ray -ti quay.io/pypa/manylinux2014_x86_64  /ray/python/build-wheel-manylinux2014.sh
+docker run -e BUILDKITE="false" -e TRAVIS_COMMIT="test" --rm -w /ray -v `pwd`:/ray -ti quay.io/pypa/manylinux2014_x86_64 /ray/python/build-wheel-manylinux2014.sh
 ```
 
 The wheel files will be placed in the .whl directory.
 
 ## Building MacOS wheels
```
Comment (glennmoy) [inline python/README-building-wheels.md:24]:
```suggestion
Running the MacOS command locally I first had to `brew install` `npm` (9.8.1), `nvm` (0.39.5),`miniconda` (23.5.2), and `python@3.8`-`python@3.10`.

```

File: python/README-building-wheels.md [hunk 2]
```diff
@@ -6,15 +6,37 @@ repository, including both changes to tracked files, and ANY untracked files.
...
+Running the MacOS command locally you must first`brew install` `npm` (9.8.1), `nvm` (0.39.5),`miniconda` (23.5.2), and `python@3.8`-`python@3.10`.
```
Comment (glennmoy) [inline python/README-building-wheels.md:24]:
```suggestion

Before unning the MacOS script you must first `brew install` `npm` (9.8.1), `nvm` (0.39.5),`miniconda` (23.5.2), and `python@3.8`-`python@3.10`.
Note: The current version of Ray has separate scripts for `macos` and `macos-arm64`. These are consolidated in a later version.
```

File: python/README-building-wheels.md [hunk 3]
```diff
@@ -6,15 +6,37 @@ repository, including both changes to tracked files, and ANY untracked files.
...
+
+Run the contents of this if-block first (or delete the conditional check):
```
Comment (glennmoy) [inline python/README-building-wheels.md:26]:
```suggestion
If buliding on `macos-64` then you must run the contents of this if-block first (or delete the conditional check):
```

**Review by kleinschmidt — COMMENTED** [review]
given that we're rebasing/upstreaming this on ray master, is this PR still needed?

**Comment by glennmoy** [comment]
> given that we're rebasing/upstreaming this on ray master, is this PR still needed?

nope - neither are any of the others really

**Pull request closed without merging**